src/main/java/org/skriptlang/skript/util/ClassLoader.java

```diff
@@ -53,20 +53,31 @@ public static void loadClasses(Class<?> source, File jarFile, String basePackage
 
 	private final String basePackage;
 	private final Collection<String> subPackages;
+	private final Collection<String> excludedPackages;
 	private final @Nullable Predicate<String> filter;
 	private final boolean initialize;
 	private final boolean deep;
 	private final @Nullable Consumer<Class<?>> forEachClass;
 
-	private ClassLoader(String basePackage, Collection<String> subPackages, @Nullable Predicate<String> filter,
-						boolean initialize, boolean deep, @Nullable Consumer<Class<?>> forEachClass) {
+	private ClassLoader(
+		String basePackage,
+		Collection<String> subPackages,
+		Collection<String> excludedPackages,
+		@Nullable Predicate<String> filter,
+		boolean initialize,
+		boolean deep,
+		@Nullable Consumer<Class<?>> forEachClass
+	) {
 		if (!basePackage.isEmpty()) { // allow empty base package
 			basePackage = basePackage.replace('.', '/') + "/";
 		}
 		this.basePackage = basePackage;
 		this.subPackages = subPackages.stream()
 				.map(subPackage -> subPackage.replace('.', '/') + "/")
 				.collect(Collectors.toSet());
+		this.excludedPackages = excludedPackages.stream()
+			.map(excluded -> excluded.replace('.',  '/') + "/")
+			.collect(Collectors.toSet());
```

Comment:
```suggestion
		this.excludedPackages = excludedPackages.stream()
				.map(excluded -> excluded.replace('.',  '/') + "/")
				.collect(Collectors.toSet());
```
could also be a helper since this is now duplicated code